chapters/request_size.rst

```diff
@@ -0,0 +1,133 @@
+=================
+Request Size
+=================
+
+``request_size`` is the parameter that we could set for the Flow.
+It defines the number of data sent into parallel ``Peas``.
```

Comment:
This is not accurate, it defines the number of `Document` a single `Request` will contain

Reply:
so every Pea will receive requests containing this amount of documents

Reply:
✅